### ISSUE TITLE
Video station will not index new videos

### ISSUE THREAD

**Issue by taquionbcn** [007revad/Video_Station_for_DSM_722#48]
After following the process of installation and update everything seems to work, I can see again videos from the web browser and from my phone.

But when I added a new set of videos to the TVshow folders they weren't indexed.

For if it was a format issue, I also tried to move already indexed videos to a new folder and create a new library but the new library n video appears as it doesn't have content.

Thank you very much.

**Comment by 007revad** [comment]
I just realised my TV Shows folder isn't showing any tv shows either.

I had to select the tv-shows folder and then it started indexing. 
![image](https://github.com/user-attachments/assets/cb1b9985-ce29-462b-9206-52e2950d1f54)

![image](https://github.com/user-attachments/assets/265a4abb-c4ad-425a-be78-284d22955fa9)

I can now see my old tv shows and newly added tv shows. Though because I already had video station open I had to click on Movies and then click on TV Shows before they appeared.
